tests/testthat/test-alpha.r

```diff
@@ -29,3 +29,18 @@ test_that("col values recycled to match alpha", {
 
   expect_equal(alphas, reds_alpha)
 })
+
+test_that("preserves element names", {
+  expect_equal(c("first", "second", "third"), names(alpha(c("first" = "gold", "second" = "lightgray", "third" = "#cd7f32"), .5)))   # quoted
```

Comment:
Again, given that all the code does is call `names<-`, I don't think these tests are necessary. You are adding additional overhead (tests that need to be read and understood) for no additional robustness.